kastenwesen.py

```diff
@@ -313,6 +324,17 @@ def rebuild(self, ignore_cache=False):
     def is_running(self):
         return False
 
+    @property
+    def is_built(self):
+        """Return True if image for this container exists locally."""
+        return any(
+            any(
+                tag == NAMESPACE + self.name + (':latest' if ':' not in self.name else '')
+                for tag in image['RepoTags']
+            )
+            for image in DOCKER_API_CLIENT.images(name=NAMESPACE + self.name)
```

Comment:
I think this docker-specific implementation actually belongs to `DockerContainer`, because `AbstractContainer` is generic.